back/src/filters/serve-static.exception.ts

```diff
@@ -0,0 +1,14 @@
+import { ArgumentsHost, Catch, ExceptionFilter, HttpException, HttpStatus } from '@nestjs/common';
+import { Response } from 'express';
+
+@Catch()
+export class ServeStaticExceptionFilter implements ExceptionFilter {
+	catch(exception: any, host: ArgumentsHost): any {
+		const ctx = host.switchToHttp();
+		const response = ctx.getResponse<Response>();
+		const status = exception instanceof HttpException ? exception.getStatus() : HttpStatus.INTERNAL_SERVER_ERROR;
+
+		if (exception.code === 'ENOENT') response.status(HttpStatus.NOT_FOUND).json(exception);
+		response.status(status).json(exception.response);
+	}
+}
```

Comment:
I'm really sorry but I have no idea what this is made for, if you could also explicit please.
(we can do that when you're here) so you don't need to type everything.

Reply:
It permits to catch 500 errors when accessing to non-existent avatar images and return a 404 error instead.

https://github.com/nestjs/serve-static/issues/139